test/functional/tests/stats/test_display_statistics.py

```diff
@@ -0,0 +1,420 @@
+        TestRun.executor.kill_process(fio_pid)
+
+
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+def test_core_config_stats():
+    """
+        title: Test CAS configuration informations for core device.
+        description: |
+          Check CAS ability to display proper configuration informations
+          in statistics for core device in every cache mode before and after IO.
+        pass_criteria:
+          - Core configuration statistics matches core properties.
+    """
+
+    with TestRun.step("Partition cache and core devices"):
+        cache_dev, core_dev = storage_prepare()
+
+    with TestRun.step(
+        f"Start {caches_count} caches (one for every cache mode) "
+        f"and add {cores_per_cache} cores per cache"
+    ):
+        caches, cores = cache_prepare(cache_dev, core_dev)
+
+    with TestRun.step(f"Get configuration statistics for each core and validate it"):
+        validate_core_config_statistics(cores)
+
+    with TestRun.step("Run 'fio'"):
+        fio = fio_prepare()
+        for i in range(caches_count):
+            for j in range(cores_per_cache):
+                fio.add_job().target(cores[i][j].system_path)
+        fio_pid = fio.run_in_background()
+
+    with TestRun.step(f"Wait {time_to_wait} seconds"):
+        time.sleep(time_to_wait)
+
+    with TestRun.step("Check core configuration statistics after IO"):
+        validate_core_config_statistics(cores)
+        failed_stats = ""
+        for i in range(caches_count):
+            cache_mode = CacheMode[caches[i].get_statistics().config_stats.write_policy.upper()]
+            cores_stats = [
+                cores[i][j].get_statistics(stat_filter=[StatsFilter.conf])
+                for j in range(cores_per_cache)
+            ]
+            if CacheModeTrait.LazyFlush in CacheMode.get_traits(cache_mode):
+                for j in range(cores_per_cache):
+                    if cores_stats[j].config_stats.dirty_for.total_seconds() <= 0:
+                        failed_stats += (
+                            f"For exported object {cores[i][j].system_path} "
+                            f"in {cache_mode} cache mode, value is "
+                            f"{cores_stats[j].config_stats.dirty_for}, should be greater then 0\n")
+            else:
+                for j in range(cores_per_cache):
+                    if cores_stats[j].config_stats.dirty_for.total_seconds() != 0:
+                        failed_stats += (
+                            f"For exported object {cores[i][j].system_path} "
+                            f"in {cache_mode} cache mode, value is "
+                            f"{cores_stats[j].config_stats.dirty_for}, should equal 0\n")
+
+        if failed_stats:
+            TestRun.LOGGER.error(
+                f"Wrong values for 'Dirty for' in core configuration "
+                f"statistics after IO:\n{failed_stats}")
```

Comment:
If you would extend `validate_core_config_statistics()` interface with `after_io` param, this check could be moved to `validate_cache_config_statistics()`

Reply:
Done